### PR TITLE
Renamed variables from single letter to better naming

### PR DESCRIPTION
**Pull request by daspartho**
Based on issue #198 
Renamed single-letter variables to more explicit variables.
- q -> query
- k -> key
- v -> value
- b -> batch
- c -> channel
- h -> height
- w -> weight
- x -> hidden_states
- x_in -> residual

@patil-suraj could you check it?
Thanks :)

### CODE REVIEW

File: src/diffusers/models/attention.py [hunk 1]
```diff
@@ -139,14 +139,14 @@ def _set_attention_slice(self, slice_size):
...
+        batch, channel, height, weight = x.shape
         x_in = x
         x = self.norm(x)
         x = self.proj_in(x)
-        x = x.permute(0, 2, 3, 1).reshape(b, h * w, c)
+        x = x.permute(0, 2, 3, 1).reshape(batch, height * weight, channel)
         for block in self.transformer_blocks:
             x = block(x, context=context)
-        x = x.reshape(b, h, w, c).permute(0, 3, 1, 2)
+        x = x.reshape(batch, height, weight, channel).permute(0, 3, 1, 2)
```
Comment (patil-suraj) [inline src/diffusers/models/attention.py:149]: here, lets rename `x` -> `hidden_states` and `x_in` ->  `residual`

File: src/diffusers/models/attention.py [hunk 2]
```diff
@@ -250,19 +250,19 @@ def reshape_batch_dim_to_heads(self, tensor):
     def forward(self, x, context=None, mask=None):
         batch_size, sequence_length, dim = x.shape
 
-        q = self.to_q(x)
+        query = self.to_q(x)
```
Comment (patil-suraj) [inline src/diffusers/models/attention.py:253]: `x` -> `hidden_states`

**Review by patil-suraj — COMMENTED** [review]
Thanks a lot, just a couple more changes!

**Comment by daspartho** [comment]
@patil-suraj renamed variable names as per suggestion :)

File: src/diffusers/models/attention.py [hunk 3]
```diff
@@ -137,18 +137,18 @@ def _set_attention_slice(self, slice_size):
...
-    def forward(self, x, context=None):
+    def forward(self, hidden_states, context=None):
         # note: if no context is given, cross-attention defaults to self-attention
-        batch, channel, height, weight = x.shape
-        x_in = x
-        x = self.norm(x)
-        x = self.proj_in(x)
-        x = x.permute(0, 2, 3, 1).reshape(batch, height * weight, channel)
+        batch, channel, height, weight = hidden_states.shape
+        residual_in = hidden_states
```
Comment (patil-suraj) [inline src/diffusers/models/attention.py]:
```suggestion
        residual = hidden_states
```

**Review by patil-suraj — APPROVED** [review]
Thanks! Merging now

**Pull request merged**